docs/view.md

```diff
@@ -609,6 +609,20 @@ Rasterization incurs an off-screen drawing pass and the bitmap consumes memory.
 
 ---
 
+### `tabIndex` <div class="label android">Android</div>
+
+Whether this `View` should be focusable with a non-touch input device, eg. receive focus with a hardware keyboard.
+Supports the following values:
+
+- `0` - View is focusable
+- `-1` - View is not focusable
+
+| Type   |
+| ------ |
+| number |
```

Comment:
```suggestion
| Type       |
| ---------- |
| enum(0, 1) |
```

For value unions we use enum notation in docs so far, however this might change in the future.

Reply:
Good call, fixed